RandomizerHost/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Data.HashFunction;
+using System.Data.HashFunction.FNV;
+using System.Text;
+
+namespace RandomizerHost.Extensions
+{
+    public static class StringExtensions
+    {
+        public static Int32 ToHash32(this String in_String)
```

Comment:
As part of the switch to `UInt32`, I think this should change to `UInt32` as well.

Reply:
See my other comment above.